cypress/e2e/court-cases/court-case-details/ExceptionHandlerPrompt.cy.ts

```diff
@@ -0,0 +1,52 @@
+import ErrorMessages from "types/ErrorMessages"
+import HO100309 from "../../../../test/test-data/HO100309.json"
+import hashedPassword from "../../../fixtures/hashedPassword"
+
+describe("ExceptionHandlerPrompt", () => {
+  before(() => {
+    cy.task("clearCourtCases")
+    cy.task("clearUsers")
+    cy.task("insertUsers", {
+      users: [
+        {
+          username: "Bichard01",
+          visibleForces: ["01"],
+          forenames: "Bichard Test User",
+          surname: "01",
+          email: "bichard01@example.com",
+          password: hashedPassword
+        }
+      ],
+      userGroups: ["B7NewUI_grp", "B7GeneralHandler_grp"]
+    })
+  })
+
+  beforeEach(() => {
+    cy.task("clearCourtCases")
+  })
+
+  context("Result qualifier - HO100309", () => {
+    it("Should display no error prompt if no exception is raised", () => {
```

Comment:
I think this makes more sense if it says _if HO100309 is not raised_

Reply:
Done